river/tree/_nodes/htr_nodes.py

```diff
@@ -65,8 +51,8 @@ def manage_memory(self, criterion, last_check_ratio, last_check_vr, last_check_e
         last_check_e
             Hoeffding bound value calculated in the last split attempt.
         """
-        for obs in self.attribute_observers.values():
-            if isinstance(obs, NumericAttributeRegressionObserver):
+        for obs in self.splitters.values():
+            if isinstance(obs, EBSTSplitter):
```

Comment:
Need to change this too.

Reply:
Changed in commit [d6931d9](https://github.com/online-ml/river/pull/488/commits/d6931d9e1632978aeb7276b9a13cff1ac2a3ec09).